packages/image_picker/image_picker_ios/example/ios/RunnerTests/ImagePickerPluginTests.m

```diff
@@ -529,11 +529,19 @@ - (void)testPickImageAuthorizationDenied API_AVAILABLE(ios(14)) {
                       quality:nil
                  fullMetadata:YES
                    completion:^(NSString *result, FlutterError *error) {
+                     // Does not error due to access denied
```

Comment:
Should we remove this test entirely as this is no longer a concern?

Reply:
I removed this test.